_pages/backup-wallet/seeds-advanced.md

```diff
@@ -74,3 +74,32 @@ Place your paper into a plastic bag with a good seal (vacuum sealed even better)
 
 #### Fireproof Document Bag
 You can use these to protect seed backups, and also to protect your physical hardware wallets.
+
+## Example Backup Configuration
+
+| Wallet | 2 of 3                        | 3 of 5                        |
+|--------|-------------------------------|-------------------------------|
+| #1     | Safe at home                  | Safe at home                  |
+| #2     | Bank or trusted friend/family | Safe at home                  |
```

Comment:
For 3-of-5 I'd sub the second "Safe at home" for "Safe at workplace or secondary residence. Obviously, not everyone has a secure work environment and/or secondary residence, but keep 2 seeds at home somewhat defeats the purpose of having a larger quorum (just stick with 2-of-3).